Realm/RLMSyncConfiguration.mm

```diff
@@ -56,6 +61,50 @@ RLMSyncSystemErrorKind errorKindForSyncError(SyncError error) {
 }
 }
 
+struct BeforeClientResetWrapper {
+    RLMClientResetBeforeBlock block;
+    RLMRealmConfiguration *rlmConfig;
+    void operator()(std::shared_ptr<Realm> local) {
```

Comment:
This is a point where we're making a C++ -> obj-c call so the body of this function needs to be wrapped in an `@autoreleasepool` or the autoreleased objects created in this function may live a very long time.

Reply:
Changed [here](https://github.com/realm/realm-swift/pull/7662/commits/1acd21cb3a162ec702f9fd49b697f2cea50ef2d8). The github UI didn't update.